item_init/automation/jsr223/python/community/item_init/300_item_init.py

```diff
@@ -53,22 +53,30 @@ def item_init(event):
 
```

Comment:
I'm ambivalent about the `300` prefix. Some users use these numbers pretty heavily. Is 300 small enough? Too small? I don't use them in my setup and haven't seen enough users who do use them any more to have a good feel for whether 300 is the right number. Do you have a feel for other people's configuration to know if 300 is the right number?

Reply:
I chose the 300 prefix as the helper libraries are from 000 to 200. So it would be loaded straight after the helpers libraries and before any other rules, as I don't use numbering for my rules.
I am new to the jython rules writing so I don't really have a feel for what people's configuration would be.

Reply:
OK, that's a good enough reasoning for why 300 is a good choice.